### PR TITLE
Fixed 'to to' to 'to'

### PR DESCRIPTION
**Pull request by Eksapsy**
Too many to's in this commit.

### CODE REVIEW

**Comment by Eksapsy** [comment]
I signed it!

File: doc/codefmt.txt [hunk 1]
```diff
@@ -70,7 +70,7 @@ The path to the google-java executable.  Generally, this should have the form:
 Default: 'google-java-format' `
 
                                                        *codefmt:shfmt_options*
-Command line arguments to to feed shfmt. Either a list or callable that takes
```
Comment (malcolmr) [inline doc/codefmt.txt]: This file is generated by running https://github.com/google/vimdoc over the source, e.g. from https://github.com/google/vim-codefmt/blob/ddc5b8/instant/flags.vim#L98. Could you fix the source and regenerate this file?

Comment (Eksapsy) [reply doc/codefmt.txt]: Yes, I could do it.

**Comment by dbarnett** [comment]
Thanks!

**Pull request merged**